content/guides/getting-started-with-php.md

```diff
@@ -0,0 +1,57 @@
+# Getting Started with PHP
+
+1. TOC
+{:toc}
+
+## Preface
+
+In this guide, we will learn how to use BookingSync API using PHP language and our dedicated [PHP lib](https://github.com/BookingSync/rest-request-php).
+
+We will here run *get_rentals* example.
+
+## Requirements
+
+1. Clone [this repository](https://github.com/BookingSync/rest-request-php) into your web server. e.g: `/var/www/`
+
+2. Create a new Vhost
+
+~~~
+<VirtualHost *:80>
+    ServerName bsa-rest-php.dev
+
+    DocumentRoot /var/www/rest-request-php/examples/api-v3/
+    <Directory /var/www/rest-request-php/examples/api-v3/>
+        AllowOverride All
+        Order Allow,Deny
+        Allow from All
+    </Directory>
+</VirtualHost>
+~~~
+
+## Create the application
+
+Applications that want to use the API need to be registered in the
+[partner panel on BookingSync.com](https://www.bookingsync.com/en/partners/applications).
+
+If you are not a partner yet, it's [free to signup so join us :)](https://www.bookingsync.com/en/partners/signup)
+
+Click on `New Application` button
+
+* Name: Choose a name for your application. e.g: `REST PHP`
+* Admin url: Useless here, choose want you want. e.g: http://google.com
```

Comment:
best to use the redirect URL as well in this case, so opening the app from BookingSync admin dashboard will run the script.